### PR TITLE
DevWs for Most Target Limited containing containing ZLL White Color Temperature Bulb

### PR DESCRIPTION
**Pull request by adurosmart**

### CODE REVIEW

**Comment by shrakuma** [comment]
@greens @MAblewiczS can you please review the FP submission

File: devicetypes/smartthings/zll-white-color-temperature-bulb.src/zll-white-color-temperature-bulb.groovy [hunk 1]
```diff
@@ -26,14 +26,12 @@ metadata {
 
 		attribute "colorName", "string"
 
-		// Generic
-		fingerprint profileId: "C05E", deviceId: "0220", inClusters: "0006, 0008, 0300", deviceJoinName: "Generic Color Temperature Light"
```
Comment (greens) [inline devicetypes/smartthings/zll-white-color-temperature-bulb.src/zll-white-color-temperature-bulb.groovy:30]: please don't delete existing fingerprints

File: devicetypes/smartthings/zll-white-color-temperature-bulb.src/zll-white-color-temperature-bulb.groovy [hunk 2]
```diff
@@ -26,14 +26,12 @@ metadata {
...
-
-		// IKEA
+		fingerprint profileId: "C05E", deviceId: "0220", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, FFFF, 0019", outClusters: "0019", manufacturer: "AduroSmart Eria", model: "ZLL-ColorTemperature", deviceJoinName: "ERIA Tunable White Light Bulb V2.1", mnmn:"SmartThings", vid: "generic-color-temperature-bulb-2200K-6500K"
+		fingerprint profileId: "C05E", deviceId: "0220", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, FFFF, 0019", outClusters: "0019", manufacturer: "ADUROLIGHT", model: "ZLL-ColorTemperature", deviceJoinName: "ERIA Tunable White Light Bulb V2.1", mnmn:"SmartThings", vid: "generic-color-temperature-bulb-2200K-6500K"
+		fingerprint profileId: "C05E", deviceId: "0220", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300, FFFF, 0019", outClusters: "0019", manufacturer: "Trust International B.V.", model: "ZLL-ColorTempera", deviceJoinName: "ERIA Tunable White Light Bulb V2.0", mnmn:"SmartThings", vid: "generic-color-temperature-bulb-2200K-6500K"
```
Comment (greens) [inline devicetypes/smartthings/zll-white-color-temperature-bulb.src/zll-white-color-temperature-bulb.groovy:32]: is this correct? `model: "ZLL-ColorTempera"`

Comment (shrakuma) [reply devicetypes/smartthings/zll-white-color-temperature-bulb.src/zll-white-color-temperature-bulb.groovy:32]: @adurosmart can you please take a look at this

**Pull request closed without merging**